### PR TITLE
Remove the padding of swiper wrapper

### PR DESCRIPTION
**Pull request by NRquadrat**
Padding no longer necessary. This provides for an incorrect calculation of the transition.

@hmerk Please test the improvment.

### CODE REVIEW

File: mainWidget/main_widget.yaml [hunk 1]
```diff
@@ -203,8 +202,8 @@ slots:
                             updateOnWindowResize: true
                             watchOverflow: true
                           style:
-                            padding-left: 50px
-                            padding-right: 50px
+                            #padding-left: 50px
+                            #padding-right: 50px
```
Comment (hmerk) [inline mainWidget/main_widget.yaml]: Please remove those lines completely, not just add a hashtag

File: mainWidget/main_widget.yaml [hunk 2]
```diff
@@ -298,8 +297,8 @@ slots:
                           style:
                             --swiper-navigation-color: var(--mW-topNavbar-swiper-navigation-color)
                             --swiper-navigation-size: var(--mW-topNavbar-swiper-navigation-size)
-                            padding-left: 50px
-                            padding-right: 50px
+                            #padding-left: 50px
+                            #padding-right: 50px
```
Comment (hmerk) [inline mainWidget/main_widget.yaml]: Same here, please remove

File: mainWidget/main_widget.yaml [hunk 3]
```diff
@@ -388,8 +387,8 @@ slots:
...
```
Comment (hmerk) [inline mainWidget/main_widget.yaml]: Same here, please remove

**Review by hmerk — CHANGES_REQUESTED** [review]

File: mainWidget/main_widget.yaml (hunk 1 above)
Comment (NRquadrat) [reply mainWidget/main_widget.yaml]: removed

File: mainWidget/main_widget.yaml (hunk 2 above)
Comment (NRquadrat) [reply mainWidget/main_widget.yaml]: removed

**Comment by NRquadrat** [comment]
Hashtags removed,
Chip for Floor reinstalled
--f7-button-text-transform: none; reinstalled

**Review by hmerk — APPROVED** [review]
LGTM

File: mainWidget/main_widget.yaml (hunk 3 above)
Comment (NRquadrat) [reply mainWidget/main_widget.yaml]: Removed

**Pull request merged**